### ISSUE TITLE
Exploring a quirkiness with the HMM workflow

### ISSUE THREAD

**Issue by meren** [meren/anvio#185]
Jeff Bowman identified a weird behavior with HMM hits. Although the HMM profiles are identical, in his installation, HMM hits are not stored in `hmm_hits_in_splits` table. We explored a couple of things already, and I thought we can continue under this issue to explore the problem.

I'm running HMMER version 3.1b2, Jeff's running 3.1b1. I'm running Prodigal version 2.6.2, and Jeff's running 2.50.

Jeff, we can use 'mini test' to look into this. Here's what I did (I was in the anvi'o codebase dorectory):

~~~ bash
$ cd tests
$ ./run_mini_test.sh # the browser should show up, close it, kill the process in the terminal w/ CTRL+C
$ anvi-get-db-table-as-matrix mini_test/test-output/ANNOTATION.db -t hmm_hits_in_splits
Database .....................................: "mini_test/test-output/ANNOTATION.db" has been initiated with its 16 tables.
Table ........................................: "hmm_hits_in_splits" has been read with 18 entries and 7 columns.
Output .......................................: hmm_hits_in_splits.txt
$ anvi-get-db-table-as-matrix mini_test/test-output/ANNOTATION.db -t hmm_hits_in_contigs
Database .....................................: "mini_test/test-output/ANNOTATION.db" has been initiated with its 16 tables.
Table ........................................: "hmm_hits_in_contigs" has been read with 8 entries and 9 columns.
Output .......................................: hmm_hits_in_contigs.txt
~~~

These two files, `hmm_hits_in_contigs.txt` and `hmm_hits_in_splits.txt` are attached. Can you check whether yours are identical?

Thank you very much for your help with this!

**Comment by bowmanjeffs** [comment]
Problem solved by updating to Prodigal 2.6.2 (Meren - let me know if you need the further exploration, I can roll back to 2.5).

**Comment by meren** [comment]
Thank you very much, Jeff. This should be enough. I will put a note in the installation manual to make sure people check for it.
